src/platform/testing/e2e/form-tester/form-filler.js

```diff
@@ -236,6 +300,7 @@ const fillForm = async (page, testData, testConfig, log) => {
       await fillPage(page, testData, testConfig, log);
 
       // Hit the continue button
+      log('Clicking continue');
```

Comment:
Kept running into a weird thing where the continue button was being clicked before I was done entering data because I was trying to call an `async` function in a `.forEach`. This log made it clear what was going on, so I decided to keep it. :man_shrugging: